internal/pkg/errors/strings.go

```diff
@@ -18,6 +18,8 @@ const (
 		StopNonInteractiveMsg + ".\n"
 	FoundNetrcCredMsg = "Found credentials for user \"%s\" from netrc file \"%s\" " +
 		StopNonInteractiveMsg + ".\n"
+	FoundKeychainCredMsg = "Found credentials for user \"%s\" from keychain " +
+		StopNonInteractiveMsg + ".\n"
```

Comment:
Does the ".\n" belong in the variable instead of outside of it?

Reply:
Following the convention here i think it belongs to the var